src/Oro/Bundle/SecurityBundle/Acl/Extension/EntityAclExtension.php

```diff
@@ -182,8 +182,8 @@ public function getAccessLevelNames($object)
      */
     public function supports($type, $id)
     {
-        if ($type === ObjectIdentityFactory::ROOT_IDENTITY_TYPE && $id === $this->getExtensionKey()) {
-            return true;
+        if ($type === ObjectIdentityFactory::ROOT_IDENTITY_TYPE) {
+            return $id === $this->getExtensionKey();
```

Comment:
That either was a bug, or is a bug now. Because logic is different from it was before.